.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,98 @@
+name: Bug report
+description: File a bug report to improve Superset's stability
+labels: ["bug"]
+body:
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: How to reproduce the bug
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
```

Comment:
In case any other reviewers are wondering, the default is false.  So in sections w/o this, they're not required.